### PR TITLE
Add version selector to the quickstart page

### PR DESCRIPTION
**Pull request by Ron-Lavi**
Selection_141

cc @lzap

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -76,25 +76,25 @@ plugins:
...
-  - 1.6
-  - 1.5
-  - 1.4
-  - 1.3
-  - 1.2
-  - 1.1
\ No newline at end of file
+  - "1.22"
+  - "1.21"
+  - "1.20"
```
Comment (Ron-Lavi) [inline _config.yml:81]: As a number the outcome is `1.2` instead of `1.20`

File: _layouts/manual.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 pagename: Manual
 latestversion: "1.22"
+url_suffix: "index"
```
Comment (lzap) [inline _layouts/manual.html]: Mind naming this "version_selector_target" instead? This feels like some global variable.

Comment (Ron-Lavi) [reply _layouts/manual.html]: updated :)

**Comment by lzap** [comment]
Nice one more thing, on the quickstart guide I see a warning _This quickstart guide is for Foreman 1.22, but 1.20 is the current stable version._ and the reason is that both are using a layout variable `layout.latestversion` which are set in two files separately. Can you drop this variable and start using `foreman_version` from the main config? This will simplify things a bit.

**Comment by lzap** [comment]
You know what, here is the patch: https://github.com/theforeman/theforeman.org/pull/1396

**Comment by Ron-Lavi** [comment]
> on the quickstart guide I see a warning This quickstart guide is for Foreman 1.22, but 1.20 is the current stable version.

Thanks ! I also noticed that the `url` is incorrect there, directing to the manuals page instead of the quickstart, fix here: https://github.com/theforeman/theforeman.org/pull/1394

**Pull request merged**